lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/test/test.factory.js

```diff
@@ -224,3 +228,127 @@ tape( 'the created function evaluates the MGF for `x` given a large range `b - a
 	}
 	t.end();
 });
+
+tape( 'the function evaluates the MGF for `x` given the case: a < c < b', function test( t ) {
```

Comment:
```suggestion
tape( 'the created function evaluates the MGF for `x` given the case: a < c < b', function test( t ) {
```

The change applies here and below and ensures consistency with the descriptions above.